books/free-programming-books-langs.md

```diff
@@ -1984,6 +1984,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Full Stack Python](https://www.fullstackpython.com) - Matt Makai
 * [Functional Programming in Python](https://www.oreilly.com/ideas/functional-programming-in-python) - David Mertz  
 * [Fundamentals of  Python Programming](https://web.archive.org/web/20191005170430/http://python.cs.southern.edu/pythonbook/pythonbook.pdf) - Richard L. Halterman (PDF) *(:construction: in process)*
+* [Geographic Data Science with Python](https://geographicdata.science/book/notebooks/01_geo_thinking.html) - Sergio Rey, Dani Arribas-Bel, Levi John Wolf (HTML)
```

Comment:
I would use https://geographicdata.science/book/intro.html as the link

Reply:
Thank you, changed link as suggested